src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.module.css

```diff
@@ -0,0 +1,63 @@
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+  padding-bottom: 100px;
+  padding-top: 100px;
+  row-gap: 50px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 80px;
+    padding-top: 40px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaLeft {
+  grid-column: 1 / span 4;
+}
+
+.gridAreaRight {
+  grid-column: 5 / span 8;
+}
+
+
+/* Logos block */
+
+.logosWrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.logoContainer {
+  height: 96.08px;
+  width: 215.14px;
```

Comment:
This probably isn't what we want. If you find yourself setting very oddly specific values (especially ones with fractional pixels), it's probably not the right approach.

I wonder if what we actually want here is another grid, as this would allow you to evenly allocate space in each row. Could you try constructing a grid with automatically generated rows, like [this example](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout#The_implicit_and_explicit_grid)?

Reply:
Yes! I agree a grid layout is a much better implementation.